File: Leetcode Cpp/Build an Array With Stack Operations.cpp
class Solution {
public:
    vector<string> buildArray(vector<int>& target, int n) {
        vector<string> res;
        int count = 1;
        for (int i = 0; i < target.size();){
            while(target[i] != count){
                res.push_back("Push");
                res.push_back("Pop");
                count++;
            }
            res.push_back("Push");
            i++;
            count++;
        }
        return res;
    }
};

File: Build an Array With Stack Operations.py
class Solution:
    def buildArray(self, target: List[int], n: int) -> List[str]:
        count=1
        z=[]

        i=0
        j=0
        while(i!=len(target)):
            if target[i]!=count:
                diff=target[i]-count
                count+=diff
                z.extend(['Push','Pop']*diff)


            z.append("Push")
            i+=1
            count+=1;
        return z